sections/sprint-1/experiment.tex

```diff
@@ -47,76 +47,116 @@ \subsection{Setup}
 \subsection{Analyzing the data}
 In \autoref{app:experiment} the data for the experiment can be seen.
 For each test the average grid was calculated, and the minimum and maximum values for $x$, $y$ and $z$ were found.
-Likewise, the average deviation was calculated from the actual grid to the average grid to get a better view of the deviation.
 The min and max values were noted, as they may be useful for future analysis.
 The $z$ values were very inconsistent throughout the entire test, and due to this and the z-axis being largely irrelevant to the game, we will not focus on analyzing them.
+Because it was decided to only calculate the deviance between the actual position and the measured position in the xy-plane.
 
 \subsubsection{Precision with 1 tag} 
-The average of all deviations is 10.54 cm.
-There were spikes where the max and min values spiked a lot.
-For \texttt{(30, 60)} the minimum $x$ value was -33.3, which is a deviation of around 60 cm and for \texttt{(120, 0)} the minimum $y$ value was -54.5 cm, which is also 54.5 cm away from the actual data point.
-There are more cases where the deviation of a single data point is far away. 
+When testing with only one tag the average of all deviations was 15.20 cm.
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=1\linewidth]{1tagdiagram.PNG}
+    \caption{A boxplot showing the dispersion of measured positions with 1 tag}
+    \label{fig:1tagdiagram}
+\end{figure}
+As can be seen on figure \ref{fig:1tagdiagram} some measurements can be quite far off the actual position. 
+On the x-axis you can see the actual position of the tag in mm compared to the playing field.
+On the y-axis is the amount in mm that the measurements were off the actual position.
+The boxplot shows the minimum and maximum values aswell as the distribution of the measurements according to what quartile it is in.
+The box is the measurements withing first and third quartile and the whiskers that are extended out from it is the measurements on outside of the these quartiles.
```

Comment:
```suggestion
The box is the measurements within the first and third quartiles and the whiskers that are extended out from it is the measurements outside of the these quartiles.
```

Reply:
ved ikke om jeg synes whiskers skal ændres. Tror det giver mere mening barre at the skrive the lines that extend